src/main/java/com/amashchenko/maven/plugin/gitflow/GitFlowReleaseFinishMojo.java

```diff
@@ -159,6 +159,14 @@ public class GitFlowReleaseFinishMojo extends AbstractGitFlowMojo {
     @Parameter(property = "useSnapshotInRelease", defaultValue = "false")
     private boolean useSnapshotInRelease;
 
+    /**
+     * Whether to skip merging into the development branch.
+     *
+     * @since 1.20.0
+     */
+    @Parameter(property = "skipMergeDevBranch", defaultValue = "false")
+    private boolean skipMergeDevBranch = false;
```

Comment:
Rename to `skipReleaseMergeDevBranch` to be inline with skipping production branch merge parameter.